accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -704,7 +709,62 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               </ol>
             </div>
 
-            <h3 id="a11y_assess">6.2 Accessibility Assessment</h3>
+            <h3 id="focus_view">6.2 Focus View</h3>
+            <div className="pa">
+              The <strong>'Focus view'</strong> switch allows you to switch between viewing all issues on the page, or only the issues for a selected element or component in the DOM. To focus on any individual element or component:
+              <ol
+                style={{ listStyleType: "decimal", marginInlineStart: "2rem" }}
+              >
+                <li>Select the element or the component in the <strong>'DOM view'</strong>, or</li>
+                <li>
+                  <strong>'Right-click'</strong> on a page element and select 'Inspect' (Chrome) or ‘Inspect Element’ (Firefox).
+                </li>
+                <li>
+                  Select the element name in the <strong>‘Focus View’</strong> switch to view only the issues for that element and its children.
+                </li>
+                <li>Select the <strong>'All'</strong> option in the <strong>‘Focus View’</strong> switch to see all issues for the page again.</li>
+              
+              By default, after the first scan of a page, all issues are shown, and the <strong>'html'</strong> element is selected, as shown in this screenshot:
```

Comment:
html should be &lt;html&gt; and doesn't need quotes or emphasis as we are talking about the actual element in the DOM, not the content of the switch component.

Reply:
agree, corrected.